### PR TITLE
Standardise casting spacing

### PR DESCRIPTION
**Pull request by timacdonald**
This PR standardises the spacing of casting in PHP via `(bool) $value` style casts.

We currently have an inconsistent usage. I feel that we should standardise on a single space, which is commonly used in the community and also helps separate the "cast" from the "value".

### CODE REVIEW

File: language/types/array.xml [hunk 1]
```diff
@@ -965,7 +965,7 @@ $error_descriptions[8] = "This is just an informal notice";
    <type>string</type>, <type>bool</type> and <type>resource</type>,
    converting a value to an <type>array</type> results in an array with a single
    element with index zero and the value of the scalar which was converted. In
-   other words, <literal>(array)$scalarValue</literal> is exactly the same as
+   other words, <literal>(array) $scalarValue</literal> is exactly the same as
```
Comment (Girgias) [inline language/types/array.xml:968]:
```suggestion
   other words, <code>(array) $scalarValue</code> is exactly the same as
```

**Review by Girgias — APPROVED** [review]
LGTM But I'm going to wait to merge for a bit as this is more a QA change and I'm not sure how behind some translations are currently

**Comment by timacdonald** [comment]
Thanks. Appreciate the reviews.

**Comment by Girgias** [comment]
Sorry for the delay, could you rebase this on the latest master?

**Comment by timacdonald** [comment]
@Girgias, no apologies needed!

I've just rebased against master here and re-checked the repository in case others had been introduced.

This should be good to go.

**Comment by Girgias** [comment]
Ah thanks for the ping, I forgot about this

**Pull request merged**